notebooks/244-depth-estimation-videpth/244-depth-estimation-videpth.ipynb

```diff
@@ -0,0 +1,1078 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    download_file('https://github.com/pronoym99/openvino_notebooks/blob/dev/notebooks/243-depth-estimation-videpth/data/image/dummy_img.png?raw=true', directory=Path(DATA_DIR / 'image'), silent=True)
</pre><p>could you please upload images on github s3 instead of links to your own repo?</p><p> </p><p>For link creation you should upload image in any comment field on github  (it is not necessary to publish comment, you can cancel it after image uploaded) and use github generated links</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/978/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
![1552097950 2672](https://github.com/openvinotoolkit/openvino_notebooks/assets/22426058/5d00b66b-57b4-4e96-91e9-36ef15ec5a0a)
![dummy_img](https://github.com/openvinotoolkit/openvino_notebooks/assets/22426058/161b9f0e-5991-4308-ba89-d81bc02bcb7c)
![1552097950 2672 (1)](https://github.com/openvinotoolkit/openvino_notebooks/assets/22426058/fc6dcc5f-f677-4618-b2ef-22e8e5cb1ebe)
![dummy_depth](https://github.com/openvinotoolkit/openvino_notebooks/assets/22426058/8c71c59f-0adf-43c6-ad13-c04431e02349)